Makefile

```diff
@@ -54,15 +54,23 @@ generate:
 
 image-aro: aro
 	docker pull registry.access.redhat.com/ubi8/ubi-minimal
-	docker build -f Dockerfile.aro -t arosvc.azurecr.io/aro:$(COMMIT) .
+	docker build -f Dockerfile.aro -t ${RP_IMAGE_ACR}.azurecr.io/aro:$(COMMIT) .
 
 image-fluentbit:
 	docker build --build-arg VERSION=1.3.9-1 \
-	  -f Dockerfile.fluentbit -t arosvc.azurecr.io/fluentbit:1.3.9-1 .
+	  -f Dockerfile.fluentbit -t ${RP_IMAGE_ACR}.azurecr.io/fluentbit:1.3.9-1 .
 
 image-proxy: proxy
 	docker pull registry.access.redhat.com/ubi8/ubi-minimal
-	docker build -f Dockerfile.proxy -t arosvc.azurecr.io/proxy:latest .
+	docker build -f Dockerfile.proxy -t ${RP_IMAGE_ACR}.azurecr.io/proxy:latest .
+
+publish-image-aro: image-aro 
+	docker push ${RP_IMAGE_ACR}.azurecr.io/aro:$(COMMIT)
+publish-image-fluentbit: image-fluentbit
```

Comment:
please add these targets (as well as image-fluentbit and image-proxy) to .PHONY (and add a newline here while you're passing?)

Reply:
done